OwenConferenciasApp/screens/formsLogin/SignInLayout.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  View,
+  TouchableOpacity,
+  TextInput,
+  Image,
+  Text,
+  StyleSheet,
+  Alert,
+} from 'react-native';
+
+export default function SignInLayout() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' ,  backgroundColor:'#F0F1F6'}}>
+      <View style={styles.containerLogin}>
+        <View>
+          <Image source={require('../../assets/logo-bc.png')} style={styles.imgLogo}></Image>
+        </View>
+        <Text style={styles.title}> BC👨‍💻 Agenda 📝</Text>
+        <TextInput
+          style={styles.inputUser}
+          placeholder="Email: "
+          autoCapitalize="none"
+        />
+        <TextInput style={styles.inputUser} placeholder="Password: " />
+        <View style={styles.containerBtns}>
+          <View style={styles.btnSignIn}>
+            <TouchableOpacity style={{width:"85%"}} onPress={() => Alert.alert('Hi')}>
+              <Text style={{color:"#ffff", textAlign:"center", fontSize:20 }} >Sign In</Text>
```

Comment:
For example, I think it would be easier to read if you wrap these styles in a single class.